pkg/network/cloud/cloud.go

```diff
@@ -0,0 +1,50 @@
+package cloud
+
+import (
+	"fmt"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
+)
+
+// Cloud is an interface that defines the cloud's use cases' operations.
+type Cloud interface {
```

Comment:
I'd suggest you rename to `CloudService` or even change the point of view and see this interface as an `Authenticator`, maybe? IMO, it's better than having the same name as the  `cloud` structure.

Reply:
IMO, adding `Service` is redundant. If we need to specify what it is, may we aren't being clear enough with context and package structure. I Agree that `Cloud` isn't good and I'll make some changes, let me know what you think.
>Heres an [example](https://github.com/mainflux/mainflux/blob/master/authn/service.go)

Reply:
Done, let me know what you think.

Reply:
I'd say `Service` can be redundant if we look to it as the example you mentioned does. In that case, it represents the interface to the service's domain capabilities (aka use cases or interactions). Instead, we want to create an interface to the **external** `users` service and that's why in clean architecture we usually place these components in the `network` or `data` directory (depending on what it provides).

What do you think about `Authenticator`? It seems to fit well in terms of interface naming best practices and we gain in flexibility to choose the struct name.

Reply:
`Authenticator` seems good, done :smile: